src/components/Todo/components/FormikMaterial/FormikMaterial.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, TextField } from '@mui/material';
+import { useFormik } from 'formik';
+import React, { FC } from 'react';
+import { IformState } from '../../../../helpers/interface';
+import { formStyles } from './formStyle';
+
+export const FormikMaterial:FC<IformState> = ({ thisInput, onSubmitTitle }) => {
+  const classes = formStyles();
+
+  const formik = useFormik({
+    initialValues: thisInput,
+    onSubmit: (values, actions) => {
+      onSubmitTitle(values);
+      actions.resetForm();
+    },
+  });
+
+  return (
+    <div className={classes.formBlock}>
+      <form className={classes.form} onSubmit={formik.handleSubmit}>
+        <TextField
+          fullWidth
+          id="title"
+          name="title"
+          label="Add new title"
+          value={formik.values.title}
+          onChange={formik.handleChange}
+          rows={4}
+        />
+        <Button type="submit" variant="contained" size="large" className="Add_Button">Save</Button>
```

Comment:
why do you write such a strange name for the class? and this button is not for adding - it's for saving